nucleus/__init__.py

```diff
@@ -51,6 +51,7 @@
     BoxAnnotation,
     CategoryAnnotation,
     CuboidAnnotation,
+    KeypointsAnnotation,
```

Comment:
we'll need to add this to `__all__` for it to start showing up in the docs ([preview here](https://scale-ai-nucleus-python-sdk--278.com.readthedocs.build/en/278/api/nucleus/index.html))